packages/core/config/webpack.common.js

```diff
@@ -3,13 +3,14 @@ const path = require('path')
 const { existsSync } = require('fs')
 
 const webpack = require('webpack')
+const Dotenv = require('dotenv-webpack')
```

Comment:
Grouped the imports so that plugins are together

Reply:
<3